presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeTpchConnectorQueries.java

```diff
@@ -39,8 +40,7 @@ public void testMissingTpchConnector()
     @Test
     public void testTpchTinyTables()
     {
-        Session session = tpchStandardTiny();
-
+        Session session = getSession();
```

Comment:
Why this change? If you use 'getSession' than you can call assertQuery without the session parameter.

Reply:
Didn't realise assertQuery was overloaded with a better method.
Fixed.